VisualStudio.gitignore

```diff
@@ -215,3 +215,15 @@ FakesAssemblies/
 **/*.Server/GeneratedArtifacts
 **/*.Server/ModelManifest.xml
 _Pvt_Extensions
+
+# Execution reports created Ranorex
+*.nlog*
+*.rxlog*
+*.rxlog
+*_trace*
+*/*.rxuser
+*.rxuser
+*.rximg
+Report
```

Comment:
I'm hesitant to include rules which cover general words like this, as it can interfere with others who may inadvertently miss this rule when committing changes.